x/emissions/module/rewards/rewards.go

```diff
@@ -184,12 +184,6 @@ func EmitRewards(ctx sdk.Context, k keeper.Keeper, blockHeight BlockHeight) erro
 	}
 	// for every topic
 	for topicId, topicReward := range topicRewards {
-		// To notify topic handler that the topic is ready for churn i.e. requests to be sent to workers and reputers
-		err = k.AddChurnReadyTopic(ctx, topicId)
```

Comment:
def gonna conflict with @guilherme-brandao 's recent PR

Reply:
will take a look, but yeah, this pr is conflicting every time I pull major-upgrade :P